README.md

```diff
@@ -35,31 +37,19 @@ The Vehicle Commander Qt demonstrates developer samples for building and testing
 
 * [New to Github? Get started here.](http://htmlpreview.github.com/?https://github.com/Esri/esri.github.com/blob/master/help/esri-getting-to-know-github.html)
 
-### Getting Started with the Vehicle Commander (Qt)
-* Check your development environment is properly configured for the ArcGIS Runtime SDK for Qt; some items of note:
-    * Ensure the configuration file, ex. `esri_runtime_qt_10_2.prf` has been installed
-        * Copied from` {ARCGISRUNTIMESDKQT}\sdk\ideintegration\esri_runtime_qt_10_2.prf to {QTDIR}\mkspecs\features\esri_runtime_qt_10_2.prf` 
-    * (Linux only) Initialize runtime. From a Command Prompt> `> . {RuntimeSDKHome}/init_sdk_qt64.sh`
-* Open, build, and run one of the sample projects in Qt Creator
-    * Note: if you are not using the latest version of the Runtime SDK, you may need to edit the .pro file for each project
-    * Change the line `CONFIG += esri_runtime_qt_10_2_2` to match the version of your Runtime SDK.
-* These projects include:
-    * ArcGISQtMessageSimulator - sends simulated network update messages
-    * ArcGISQtSearchAPIApp - a stand-alone symbol search sample
-    * MapTestApp - a simple map demonstration application
-    * VehicleCommander - a more advanced in-vehicle, touchscreen application
-        * To automate the copy of the deployment files, a manual step is required to the Qt Creator Project. To do this in Qt Creator: Projects | VehicleCommander - Build Settings tab | Build Steps click on Add Build Step | Choose Make from the Add menu | Add "install" as Make argument. See the [VehicleCommander.pro](source/VehicleCommander/VehicleCommander.pro) for details on the install step. Note: this must been done for both the Debug and Release Builds.
-        * IMPORTANT/WORKAROUND: in order to debug the VehicleCommander project you will need to disable QML debugging (in Projects | Run | Debugger Settings). For more information see [Issue #5](https://github.com/Esri/vehicle-commander-qt/issues/5)
-
-### Services
-
-* The default application uses ArcGIS Online services to display basemaps. See the source code for more details.
+### Getting Started with Message Simulator
+#### Building from Source
+* Fork and clone this repo.
+* Use Qt Creator to open source/ArcGISQtMessageSimulator/ArcGISQtMessageSimulator.pro.
+* Run the application and run the simulation with one of the sample message files in source/ArcGISQtMessageSimulator/SampleMessages.
+* Confirm that Vehicle Commander or Squad Leader displays the messages. Keep in mind that the computer running Message Simulator and the computer running Vehicle Commander or the device running Squad Leader must be connected to the same router and that router must allow UDP traffic. Alternatively, you could use Qt to develop a simple client app that receives UDP messages, maybe using [QUdpSocket](http://doc.qt.io/qt-5/qudpsocket.html).
```

Comment:
This is a pretty important point that someone might miss here perhaps move to another section/line and highlight/emphasize somehow ("IMPORTANT") - the PC/network must allow both UDP and broadcast UDP.